mitmproxy/contentviews/base.py

```diff
@@ -1,13 +1,23 @@
-# Default view cutoff *in lines*
-
-from typing import Iterable, AnyStr, List
+import traceback
+from typing import AnyStr
+from typing import Iterable
+from typing import List
 from typing import Mapping
+from typing import Optional
 from typing import Tuple
 
-VIEW_CUTOFF = 512
+from mitmproxy import exceptions
+from mitmproxy.net import http
+from mitmproxy.utils import strutils
+
 
+VIEW_CUTOFF = 512
 KEY_MAX = 30
 
+views = []  # type: List[View]
+content_types_map = {}  # type: Dict[str, List[View]]
+view_prompts = []  # type: List[Tuple[str, str]]
```

Comment:
Not sure if this belongs together. `base` previously was stuff that was used by views only, whereas `__init__` contained the external API. Maybe move it to `viewmanager` or so?